examples/apps/weather/src/Error.flix

```diff
@@ -0,0 +1,26 @@
+mod WeatherError {
+    use Json.JsonError
+
+    ///
+    /// All sorts of errors we may encounter during the request
+    ///
+    pub enum WeatherError {
```

Comment:
This should be one level out. Otherwise you have WeatherError.WeatherError -- which is nonsense.